src/utils/translate/en.json

```diff
@@ -145,7 +145,16 @@
   "DropdownQuestions": {
     "Question1": {
       "Question": "Where does SeroTracker's data come from?",
-      "Answer": "We collect data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). The sources included in our dashboard to date are in the table below."
+      "Answer": {
+        "Part1": "We collect all seroprevalence data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). Data related to vaccination status and case counts is pulled from the",
+        "Part2": "The sources included in our dashboard to date are in the table below."
+      },
+      "Links": {
```

Comment:
Nit: this doesn't need to be in a translations file

Reply:
wait which part doesn't need to be in translations file?

Reply:
The links, as URLs are language agnostic

Reply:
I think having links here is actually helpful. Looking at the translate function for the FAQ questions the links are in the middle of answers, so either we have links there or we hardcode the links with multiple translate functions in the code file. I think either works.